CMakeLists.txt

```diff
@@ -44,15 +42,11 @@ if(NOT NO_TRANSLATION)
 endif()
 
 # make common
-if(common_libs)
-	add_subdirectory(common/src/Utilities)
-	add_subdirectory(common/src/x86emitter)
-endif()
+add_subdirectory(common/src/Utilities)
```

Comment:
We should think about breaking Common into it's own sub project with a CMakeLists

Reply:
If we did it would literally be one file with the content
```cmake
add_subdirectory(src/Utilities)
add_subdirectory(src/x86emitter)
```

Reply:
Not nessacarily. I meant for Common having a list of each source and header so it's 

`
set(sources 
src/Utilities/path.h
)
`

Reply:
It already does, in a `CMakeLists.txt` in the directory referenced by the `add_subdirectory`
How do you think the `add_subdirectory` works?  Magic?